cmd/credentialUtil.go

```diff
@@ -138,7 +129,7 @@ func GetOAuthTokenManagerInstance() (*common.UserOAuthTokenManager, error) {
 			return
 		}
 
-		lca.persistToken = false
+		lca.persistToken = true
```

Comment:
Why are we changing behavior to start persisting tokens on auto login?

Reply:
Done in the latest commit.